citadel/install_windows_src.md

```diff
@@ -138,4 +138,4 @@ the results you want:
 
 ## Troubleshooting
 
-See [Troubleshooting](troubleshooting.md#windows)
+See [Troubleshooting](/docs/citadel/troubleshooting#windows)
```

Comment:
Same as above, maybe we want to just remove `#windows` since that section doesn't exist in Citadel.